app/views/foreman_ansible/job_templates/ansible_-_inventory.erb

```diff
@@ -0,0 +1,135 @@
+  input_type: user
+  description: Whether should be Facts included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+model: ReportTemplate
+require:
+- plugin: foreman_ansible
+  version: '>= 3.1.0'
+- plugin: katello
+￼  version: '>= 3.9.0'
+-%>
+<%- content_facet_attr = input('Content Facet Attributes') == 'yes' ? true : false -%>
+<%- opts = content_facet_attr ? { includes: [:content_facet] } : {} -%>
+<%- load_hosts(opts).each_record do |host| -%>
+<%- content_facet = host_content_facet(host) if content_facet_attr -%>
+<%- content_facet_attribute_data = nil -%>
+<%- if content_facet -%>
+<%- content_facet_attribute_data = {
+      'applicable_module_stream_count': content_facet.applicable_module_stream_count,
+      'applicable_package_count': content_facet.applicable_rpm_count,
+      'content_source': content_facet.content_source,
+      'content_source_id': content_facet.content_source_id,
+      'content_source_name': content_facet.content_source_name,
+      'content_view': {
+          'id': content_facet.content_view_id,
+          'name': content_facet.content_view_name,
+      },
+      'content_view_id': content_facet.content_view_id,
+      'content_view_name': content_facet.content_view_name,
+      'errata_counts': content_facet.errata_counts,
+      'id': content_facet.id,
+      'kickstart_repository': content_facet.kickstart_repository,
+      'kickstart_repository_id': content_facet.kickstart_repository_id,
+      'kickstart_repository_name': content_facet.kickstart_repository_name,
+      'lifecycle_environment': {
+          'id': content_facet.lifecycle_environment_id,
+          'name': content_facet.lifecycle_environment_name,
+      },
+      'lifecycle_environment_id': content_facet.lifecycle_environment_id,
+      'lifecycle_environment_name': content_facet.lifecycle_environment_name,
+      'upgradable_module_stream_count': content_facet.upgradable_module_stream_count,
+      'upgradable_package_count': content_facet.upgradable_rpm_count,
+      'uuid': content_facet.uuid,
+}
+-%>
+<%- end -%>
+<%- inventory_data = {
+      'Name': host.name,
+      'Organization': input('Organization') == 'yes' ? host.organization : nil,
+      'Location': input('Location') == 'yes' ? host.location : nil,
+      'Host Group': input('Host Group') == 'yes' && host.hostgroup ? host.hostgroup.name : nil,
+      'Host Collections': input('Host Collections') == 'yes' && host.host_collections ? host.host_collections.map { |c| c.name } : nil,
+      'IPv4': input('IPv4') == 'yes' ? host.ip : nil,
+      'Subnet': input('Subnet') == 'yes' ? host.subnet : nil,
+      'Smart Proxies': input('Smart Proxies') == 'yes' && host.smart_proxies ? host.smart_proxies.map { |p| p.name } : nil,
+      'Content Facet Attributes': content_facet_attribute_data,
+      'Facts': input('Facts') == 'yes' ? host.facts : nil
+    }
+    report_row(inventory_data)
+-%>
+<%- end -%>
+<%= report_render(format: :json) -%>
```

Comment:
Can we just keep report_render here and specify the format in the ansible inventory endpoint when we call renderer? That way people could use the same template to get the same inventory in different format.